13-TeV-examples/uproot_python/ttbar_analysis.ipynb

```diff
@@ -0,0 +1,1236 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": [],
+      "authorship_tag": "ABX9TyOMh376Wuh8DiGF9dFooGVd",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/IoPapadopoulos/ttbar_analysis/blob/main/ttbar_analysis.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "**Download data sets**\n",
+        "\n",
+        "The following cell downloads all the data.\n",
+        "\n",
+        "If you download the data, run the analysis with local_or_online = 1\n",
+        "\n",
+        "\n",
+        "\n",
+        "\n",
+        "(For colab)"
+      ],
+      "metadata": {
+        "id": "rcsbAQAmiLX_"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def download(url):\n",
```

Comment:
An `import requests` might be missing here?
I see that the package is imported in a cell below, but hot here.